docs/types.propDefinition.md

```diff
@@ -23,40 +23,46 @@ observable property.  These behaviors can be specified with as an `Object`, `Str
 }
 ```
 
-    @option {can-define.types.default} default Specifies the initial value of the property or
-    a function that returns the initial value.
+    @option {can-define.types.default} default Specifies the initial value of the property or a function that returns the initial value.
 
     ```js
+import { DefineMap } from "can";
+
 // A default age of `0`:
 const Person = DefineMap.extend( {
-	age: {
-		value: 0
-	},
-	address: {
-		value: function() {
-			return { city: "Chicago", state: "IL" };
-		}
-	}
+    age: {
+        value: 0
+    },
+    address: {
+        value: function() {
```

Comment:
I think this should be `default: function() { ... }`.

Reply:
Fixed in 1dd11b3.